### PR TITLE
Change the default answer test unit system to code

### PR DESCRIPTION
**Pull request by matthewturk**
This is a test PR to see if we get any failures that are not *answer value* failures if we change the base unit system for answer tests to `code` universally.

If this works, I will update this with a bit more of the rationale, and I will post to the yt mailing list.  It is not mergable as-is.

### CODE REVIEW

**Comment by matthewturk** [comment]
It looks like *most* of the answers are wrong in ways we expect them to be wrong.  The Ramses answers are slightly confusing, as I *think* they might be showing different array lengths.

**Comment by matthewturk** [comment]
It's also worth mentioning that this only applies to when a string is fed into things like `small_patch_amr`, so more work will need to be done, and this PR will require iteration as I turn some of these on.

**Comment by matthewturk** [comment]
I verified that the RAMSES tests are in fact doing the right thing.  So in the next update, I will be upping the answer test IDs for the affected tests.

**Comment by Xarthisius** [comment]
@yt-fido test this please

**Comment by matthewturk** [comment]
I have verified that the enzo tests are all still correct -- just changed the unit system -- and that the RAMSES tests are a result of the `weight_field` being unitless.  This commit updates the RAMSES answer test and enzo answer test items.

File: yt/utilities/answer_testing/framework.py [hunk 1]
```diff
@@ -1047,8 +1061,10 @@ def sph_answer(ds, ds_str_repr, ds_nparticles, fields):
                     yield PixelizedProjectionValuesTest(
                         ds, axis, field, weight_field,
                         dobj_name)
+            # See the other note about unit_system
```
Comment (munkm) [inline yt/utilities/answer_testing/framework.py]: maybe add a line number so it's clear what note you're referring to.

**Pull request merged**